File: week2/hw2.py
def avg(data):
    ppl_count = data["count"]
    employee = data["employees"]
    i = 0
    money = 0

    while i < ppl_count:
        money += employee[i]["salary"]
        i += 1
    print(money/ppl_count)


avg({
    "count": 3, "employees": [
        {
            "name": "John",
            "salary": 30000},
        {
            "name": "Bob",
            "salary": 60000},
        {
            "name": "Jenny",
            "salary": 50000}
    ]
})

File: week2/hw2.js
function avg(data) {
  let number = data.count
  let employees = data.employees
  let sum = 0
  for (i = 0; i < number; i++) {
    sum += employees[i].salary;
  }
  let ans = sum / number
  console.log(ans)
  return (ans)
}
avg({
  "count": 3,
  "employees": [
    {
      "name": "John",
      "salary": 30000
    },
    {
      "name": "Bob",
      "salary": 60000
    },
    {
      "name": "Jenny",
      "salary": 50000
    }
  ]
})


